articles/cli.md

```diff
@@ -0,0 +1,178 @@
+## Bonsai Command-Line Interface (CLI)
+
+Bonsai CLI (Command-Line Interface) allows workflows to be launched from the OS command-line tool. In addition to opening workflows with the default graphic interface, the CLI opens the possibility to use other advanced features, that will be covered ahead, such as "no-editor" mode, initialization of variable's values, and specifying layouts.
+
+
+&nbsp;
+___
+### Installation and basic use
+Bonsai CLI can be called by targeting ```Bonsai.exe``` executable. To make sure you are able to call ```Bonsai``` from the command line:
+
+```cmd
+> pathToBonsaiExe\Bonsai.exe
+```
+Alternatively, by adding ```pathToBonsaiExe``` to your System Path, simply:
```

Comment:
Do you mean that you can simply call Bonsai from command line after adding ```pathToBonsaiExe``` to the System Path variable?

Reply:
Yes, if the directory ```pathToBonsaiExe``` exists in windows path, it will find Bonsai.exe and call it when you simply type "Bonsai" in the command line.